src/scheduler/job_info.c

```diff
@@ -4955,16 +4955,16 @@ nspec **create_res_released_array(status *policy, resource_resv *resresv)
 	int i = 0;
 	resource_req *req;
 
-	if ((resresv == NULL) || (resresv -> nspec_arr == NULL) || (resresv -> ninfo_arr == NULL))
+	if ((resresv == NULL) || (resresv->nspec_arr == NULL) || (resresv->ninfo_arr == NULL))
 		return NULL;
 
-	nspec_arr = dup_nspecs(resresv -> nspec_arr, resresv -> ninfo_arr);
+	nspec_arr = dup_nspecs(resresv->nspec_arr, resresv->ninfo_arr);
 	if (nspec_arr == NULL)
 		return NULL;
-	for (i = 0; nspec_arr[i] != NULL; i++) {
-		for (req = nspec_arr[i] ->resreq; req != NULL; req = req -> next) {
-			if(policy->rel_on_susp != NULL) {
-				if( req -> type.is_consumable == 1 && resdef_exists_in_array(policy->rel_on_susp, req->def) ==  0)
+	if (policy->rel_on_susp != NULL) {
+		for (i = 0; nspec_arr[i] != NULL; i++) {
+			for (req = nspec_arr[i] ->resreq; req != NULL; req = req->next) {
```

Comment:
You did a great job of fixing most of the whitespaces, but there is one more... :)
Please remove the whitespace between [i] ->